<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SAME_ST_FAMILY</class>
 <widget class="QWidget" name="SAME_ST_FAMILY">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>286</width>
    <height>60</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="label_line_nm">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_st_nm">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButton_manage">
     <property name="text">
      <string>管理</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButton_del">
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
